packages/core/src/types.ts

```diff
@@ -228,7 +228,7 @@ export type RecursiveDesignTokenDefinition = {
 };
 
 export type DesignTokensDefinition = {
-  spacing?: Record<string, string>;
+  spacing?: Record<string, string> | Record<string, number>;
```

Comment:
This would also work here:
```suggestion
  spacing?: Record<string, string | number>;
```

Reply:
That is what I initially tried but you get this in the build

```
  [!] (plugin typescript) RollupError: @rollup/plugin-typescript TS2345: Argument of type 'Record<string, string | number>' is not assignable to parameter of type 'Record<string, string>'.
```